tests/test_pipelines.py

```diff
@@ -508,6 +508,18 @@ def test_set_scheduler_consistency(self):
 
         assert dict(ddim_config) == dict(ddim_config_2)
 
+    def test_save_safe_serialization(self):
+        pipeline = StableDiffusionPipeline.from_pretrained("hf-internal-testing/tiny-stable-diffusion-torch")
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            tmpdirname = "/tmp/test_save_safe_serialization"  # TODO: remove
+            pipeline.save_pretrained(tmpdirname, safe_serialization=True)
```

Comment:
Should we maybe verify the type of files that have been saved ?

Reply:
Good point, fixed in 8c12a559c399e87e31f853701e31bb8e634ca7cf